### PR TITLE
Ac employee controller list & detail views

### PR DESCRIPTION
**Pull request by asiamcarter**
# Description
Added functional list and details views to the employee controller. Updated ReadMe.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing Instructions

- Git fetch -all
- Git checkout ac-employeecontroller
- Dotnet run
- Click on 'Employees' link in navigation bar
- Confirm that a list of employees populates. Each employee entry should include the following:
1. First Name
2. Last Name
3. Department Name
- Click on an individual employee within the employee list
- Confirm that you are re-directed to the details view of an individual employee. This view should include the following information:
1. First Name
2. Last Name
3. Department Name
4. Computer Make
5. A List of training programs that the employee has attended or plans to attend

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added test instructions that prove my fix is effective or that my feature works```

### CODE REVIEW

**Review by THATBLADEBOY — APPROVED** [review]
Works as described 👍🏻

**Review by jdbbaugh — APPROVED** [review]
Hecka Rad🏆

File: BangazonWorkForceManagement/Controllers/EmployeesController.cs [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        public ActionResult Details(int id)
+        {
+            using (SqlConnection conn = Connection)
+
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @" 
+										SELECT e.FirstName AS EmployeeFirstName,
```
Comment (askingalot) [inline BangazonWorkForceManagement/Controllers/EmployeesController.cs:86]: formatting

**Review by askingalot — CHANGES_REQUESTED** [review]

File: BangazonWorkForceManagement/Controllers/EmployeesController.cs (hunk 1 above)
Comment (asiamcarter) [reply BangazonWorkForceManagement/Controllers/EmployeesController.cs:86]: corrected!

Comment (askingalot) [inline BangazonWorkForceManagement/Controllers/EmployeesController.cs:86]: I *thought* I'd left this comment earlier, but now I don't see it. I'm sorry.

How do you know that your query will return the employee's current computer?

**Review by askingalot — CHANGES_REQUESTED** [review]

File: BangazonWorkForceManagement/Controllers/EmployeesController.cs (hunk 1 above)
Comment (asiamcarter) [reply BangazonWorkForceManagement/Controllers/EmployeesController.cs:86]: Corrected!

**Review by askingalot — APPROVED** [review]

**Pull request merged**